WaveToyX/par/standing.par

```diff
@@ -13,9 +13,9 @@ Cactus::presync_mode = "mixed-error"
 
 CarpetX::poison_undefined_values = no
 
-CarpetX::periodic_x = yes
-CarpetX::periodic_y = yes
-CarpetX::periodic_z = yes
+Driver::periodic_x = yes
```

Comment:
This seems wrong; `Driver::periodic` is not set.

Reply:
yes, this should be `Driver::periodic = yes`. Hmm, I thought the tests had all passed. Will check.

Reply:
update, is not a test so nothing failed